exercises/protein-translation/README.md

```diff
@@ -0,0 +1,79 @@
+# Rna Transcription
```

Comment:
this is the description for the RNA transcription exercise (https://github.com/exercism/problem-specifications/blob/master/exercises/rna-transcription/description.md), whereas given that this is in the `protein-translation` directory it needs to be the description of the protein translation exercise: https://github.com/exercism/problem-specifications/blob/master/exercises/protein-translation/description.md